class.jetpack-post-images.php

```diff
@@ -310,18 +310,20 @@ static function from_html( $html_or_id ) {
 	 * @return Array containing details of the image, or empty array if none.
 	 */
 	static function from_blavatar( $post_id, $size = 96 ) {
-		if ( !function_exists( 'blavatar_domain' ) || !function_exists( 'blavatar_exists' ) || !function_exists( 'blavatar_url' ) ) {
-			return array();
-		}
 
 		$permalink = get_permalink( $post_id );
-		$domain = blavatar_domain( $permalink );
 
-		if ( !blavatar_exists( $domain ) ) {
-			return array();
-		}
+		if ( function_exists( 'jetpack_has_site_icon' ) && jetpack_has_site_icon() ) {
+			$url = jetpack_site_icon_url( null, $size, $default = false );
+		} elseif ( function_exists( 'blavatar_domain' ) && function_exists( 'blavatar_exists' ) && function_exists( 'blavatar_url' ) ) {
+				$domain = blavatar_domain( $permalink );
 
-		$url = blavatar_url( $domain, 'img', $size );
+				if ( ! blavatar_exists( $domain ) ) {
+					return array();
+			}
```

Comment:
I'm going to win the award for nit-pickiest nit-picker, @chaselivingston 

Lines 319, 321 and 322 should be pulled back, they're tabbed in too far.  319 & 321 should be the same indent as 317, and 322 should be one tab further than that :)